### DISCUSSION TITLE
When you plan to update the NuGet package?

### DISCUSSION THREAD

**Discussion by Schaukelking** [stephanstapel/ZUGFeRD-csharp#201] [Q&A]
Hello!

I just wanted to ask you when you plan to update the NuGet package? You already implemented some new great fixes that would help us a lot. 

Best regards and thanks for taking the time!

**Comment by Syntaxrabbit** [comment]
As of now, the sample code does not work with the current nuget release. :-\

**Comment by stephanstapel** [comment]
Hi,
I am ready to go. If you have the chance, it'd be great if you could do some testing of the latest code, especially by processing files from other sources or by processing files in other systems.

**Comment by Syntaxrabbit** [comment]
https://github.com/stephanstapel/ZUGFeRD-csharp/pull/385 :-)

**Comment by Syntaxrabbit** [comment]
https://github.com/stephanstapel/ZUGFeRD-csharp/pull/393 :-)
